onboarding-enabler-spring/src/test/java/com.ca.apiml.enable/register/ApimlDisabledRegisterToApiLayerTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.apiml.enable.register;
+
+import com.ca.apiml.enable.EnableApiDiscovery;
+import com.ca.apiml.enable.config.EnableApiDiscoveryConfig;
+import com.ca.mfaas.eurekaservice.client.ApiMediationClient;
+import com.ca.mfaas.eurekaservice.client.config.ApiMediationServiceConfig;
+import com.ca.mfaas.exception.ServiceDefinitionException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.ConfigFileApplicationContextInitializer;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+
+@RunWith(SpringRunner.class)
+@EnableAutoConfiguration
+@EnableApiDiscovery
+@ActiveProfiles("apiml-disabled")
+@ContextConfiguration(initializers = ConfigFileApplicationContextInitializer.class, classes = {RegisterToApiLayer.class, EnableApiDiscoveryConfig.class})
+public class ApimlDisabledRegisterToApiLayerTest {
```

Comment:
```java
@RunWith(SpringRunner.class)
@EnableAutoConfiguration
@EnableApiDiscovery
@ContextConfiguration(initializers = ConfigFileApplicationContextInitializer.class)
@TestPropertySource(properties = {
    "apiml.enabled = false"
})
```
It is just one parameter for this test suite. We can override it instead of having ```apiml-disabled``` profile